### PR TITLE
instant-id: added deleting of full precision pipeline before quantization

### PR DESCRIPTION
**Pull request by nikita-savelyevv**
**Changes**:
- In order to reduce peak memory footprint, full precision OV pipeline is now deleted right after calibration dataset is collected. This way it does not take up additional memory during quantization.
- Comparing inference speed of original and optimized pipelines is now optional (disabled by default) for the same reason.

After applying the changes and updating to openvino-nightly, the peak memory is observed to drop from 120 GB to 60 GB.

Related ticket: 146016

### CODE REVIEW

**Comment by nikita-savelyevv** [comment]
@l-bat , it may be beneficial to consider logic like this when implementing quantization of large pipelines in the future

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
@nikita-savelyevv please fix code formatting

File: notebooks/instant-id/instant-id.ipynb [hunk 1]
```diff
@@ -1810,43 +1810,6 @@
     "device"
```
Comment (eaidova) [inline notebooks/instant-id/instant-id.ipynb:1810]: <pre class="ql-syntax" spellcheck="false">Line #4.    def create_ov_pipe():
</pre><p>maybe better to allow specify paths for reusing the same way of pipeline creation for int8 pipeline too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2539/discussion/#comment-1849850461'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='36'/>

Comment (nikita-savelyevv) [reply notebooks/instant-id/instant-id.ipynb:1810]: Done

**Pull request merged**